tests/unit_tests/bittensor_tests/utils/test_utils.py

```diff
@@ -235,17 +235,29 @@ def test_is_valid_ed25519_pubkey():
     assert not bittensor.utils.is_valid_ed25519_pubkey(bad_pubkey)
 
 def test_registration_diff_pack_unpack():
-        fake_diff = pow(2, 31)# this is under 32 bits
-        
-        mock_diff = multiprocessing.Array('Q', [0, 0], lock=True) # [high, low]
-        
-        bittensor.utils.registration_diff_pack(fake_diff, mock_diff)
-        assert bittensor.utils.registration_diff_unpack(mock_diff) == fake_diff
-
-        fake_diff = pow(2, 32) * pow(2, 4) # this should be too large if the bit shift is wrong (32 + 4 bits)
-        
-        bittensor.utils.registration_diff_pack(fake_diff, mock_diff)
-        assert bittensor.utils.registration_diff_unpack(mock_diff) == fake_diff
+    fake_diff = pow(2, 31)# this is under 32 bits
+    
+    mock_diff = multiprocessing.Array('Q', [0, 0], lock=True) # [high, low]
+    
+    bittensor.utils.registration_diff_pack(fake_diff, mock_diff)
+    assert bittensor.utils.registration_diff_unpack(mock_diff) == fake_diff
+
+    fake_diff = pow(2, 32) * pow(2, 4) # this should be too large if the bit shift is wrong (32 + 4 bits)
+    
+    bittensor.utils.registration_diff_pack(fake_diff, mock_diff)
+    assert bittensor.utils.registration_diff_unpack(mock_diff) == fake_diff
```

Comment:
These 2 asserts covers the same scenario? or are they from different scenarios??

From xUnit principles, it's a good practice to keep assertions group by scenario/usecase, so, if one group of aserts fail we must know what's the failure about (a group could be of one in this example).

If they are identifying different cases, i would split them in two test functions.

Reply:
Good point. I'll split these. One tests less than 32 bits, the other more than 32 bits